content/en/docs/quickstart.md

```diff
@@ -26,6 +26,17 @@ docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.3
 >
 > - Since `v1.0.0-rc.1` release, by default, Notation stores signatures using [OCI Artifact Manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/artifact.md), which is defined in [OCI Image spec v1.1.0](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc2)). If you choose a different registry, make sure the registry is compatible with OCI Image spec v1.1.0.
 
+### Building OCI-compatible registry image from source
```

Comment:
Would it be better to mark it as optional? Such as "Building OCI-compatible registry image from source (optional)"